contracts/modules/staking/StakingPool.sol

```diff
@@ -1468,26 +1468,60 @@ contract StakingPool is IStakingPool, ERC721 {
     uint totalCapacity,
     uint globalMinPrice,
     bool useFixedPrice
-  ) internal returns (uint) {
+  ) internal returns (uint premium) {
 
     StakedProduct memory product = products[productId];
     uint targetPrice = Math.max(product.targetPrice, globalMinPrice);
```

Comment:
i could argue for putting in this max() calculation inside the pure function as well and pass in globalMinPrice

but it's trivial, so this looks great as well